pdfbox/pom.xml

```diff
@@ -75,6 +75,12 @@
             <artifactId>jbig2-imageio</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.mockito</groupId>
+            <artifactId>mockito-core</artifactId>
```

Comment:
I didn't see any existing tests using a mocking library.  I'm not sure if this is intentional.  I'm also not sure if pdfbox maintainers have a policy against adding new dependencies.  If mockito is undesirable for some reason, I'll have to come up with a different way to test my proposed code change.